2018 Updates

ewdurbin opened the pull request:
Mostly a port of @sagesharp's work to Markdown

ewdurbin committed:
2018 Updates, mostly a port of @sagesharp's work to Markdown

ewdurbin commented on code_of_conduct.md:
```diff
@@ -1,57 +1,78 @@
-We value the participation of each member of the Python community and want all attendees to have an enjoyable and fulfilling experience. Accordingly, all attendees are expected to show respect and courtesy to other attendees throughout the conference and at all conference events, whether officially sponsored by PyCon or not.
+We value the participation of each member of the Python community and want all attendees to have an enjoyable and fulfilling experience. Accordingly, all attendees are expected to show respect and courtesy to other attendees throughout the conference and at all conference events, whether officially sponsored by the Python Software Foundation or not.
 
 To make clear what is expected, all delegates/attendees, speakers, exhibitors, organizers and volunteers at any PyCon event are required to conform to the following Code of Conduct. Organizers will enforce this code throughout the event.
```
It's unclear to me what `delegates` mean in this context.

sagesharp replied:
We can remove the word if it doesn't make sense. Perhaps "staff" instead of delegates?

ewdurbin commented on code_of_conduct.md:
```diff
@@ -1,57 +1,78 @@
+PyCon is dedicated to providing a harassment-free conference experience for everyone, regardless of age, gender identity and expression, sexual orientation, disability, physical appearance, body size, ethnicity, nationality, race, or religion (or lack thereof), education, or socio-economic status.
 
-All communication should be appropriate for a professional audience including people of many different backgrounds. Sexual language and imagery is not appropriate for any conference venue, including talks.
+Examples of behavior that contributes to creating a positive environment include:
```
@sagesharp Do we have any good source material for adding a few more points to this list? It feels a tad imbalanced against the following list.

sagesharp replied:
Sure! I only left in the positive examples that were already in the document.

The [Contributor Covenant](https://www.contributor-covenant.org/version/1/4/code-of-conduct) has some short social guidelines:
 * Using welcoming and inclusive language
 * Being respectful of differing viewpoints and experiences
 * Gracefully accepting constructive criticism
 * Focusing on what is best for the community
 * Showing empathy towards other community members

The [Django Project](https://www.djangoproject.com/conduct/) has some more detailed social guidelines, but most of them are pretty well summarized by the above points.

The [Recurse Center](https://www.recurse.com/code-of-conduct) has a set of social rules that are geared towards being very newcomer-friendly:
 * **No feigning surprise.** The first rule means you shouldn't act surprised when people say they don't know something. This applies to both technical things ("What?! I can't believe you don't know what the stack is!") and non-technical things ("You don't know who RMS is?!"). Feigning surprise has absolutely no social or educational benefit: When people feign surprise, it's usually to make them feel better about themselves and others feel worse. And even when that's not the intention, it's almost always the effect. As you've probably already guessed, this rule is tightly coupled to our belief in the importance of people feeling comfortable saying "I don't know" and "I don't understand."
 * **No well-actually's.** A well-actually happens when someone says something that's almost - but not entirely - correct, and you say, "well, actually…" and then give a minor correction. This is especially annoying when the correction has no bearing on the actual conversation. This doesn't mean we aren't interested in truth-seeking or that we don't care about being precise. Almost all well-actually's in our experience are about grandstanding, not truth-seeking. (Thanks to Miguel de Icaza for originally coining the term "well-actually.")

Depending on how newcomer-friendly you want the PyCon sprints to be, you might consider adopting the two rules from the Recurse Center. I would encourage adopting the bullet points from the Contributor Covenant at the very least.

ewdurbin replied:
Thank you @sagesharp! The Recurse Center social rules are of my new favorites (I recently attended a batch). I think they are _strongly_ effective, but am concerned that they do take a little bit more introduction than we can reasonably allot.

I'll add the Contributor Covenant points, and we could consider the Recurse Center's social rules for inclusion in time.

ewdurbin committed:
update, add links and formatting

ewdurbin committed:
names, titles, contact information

ejodlowska reviewed (commented):
<tbd> email can be changed to pycon-us-report@python.org

sagesharp commented on Staff Procedure for incident handling.md:
```diff
@@ -1,65 +1,122 @@
- - Are we doing anything about it?
- - Who is doing those things?
- - When are they doing them?
+Give reported person a place to appeal to if there is one. Keep in mind that it is not a good idea to encourage an apology from the harasser.
```
"Give reported person a place to appeal to if there is one"
should be
"If a reported person wants to appeal the decision, they may contact the Python Software Foundation Board <psf@python.org>."

"Keep in mind that it is not a good idea to encourage an apology from the **harasser**."
should be
"Keep in mind that it is not a good idea to encourage an apology from the **reported person**."

sagesharp commented on Attendee Procedure for incident handling.md:
```diff
@@ -1,38 +1,64 @@
+
+## Report Handling Procedure
+
+When you make a report to an incident responder, they will gather information about the incident according to the PyCon US [Staff Procedure For Incident Response] (https://us.pycon.org/2018/about/code-of-conduct/staff-procedure/).
```
Space in between the link syntax here.

ewdurbin committed:
address review

- further generalize harassment usage
- add more positive behaviors
- update links/phone numbers

ewdurbin committed:
specify conflict of interest avoidance in Attendee procedure

ewdurbin committed:
Markdown

ewdurbin committed:
no need for telephone # in this place

ewdurbin committed:
update notice/bylines

ewdurbin commented:
@ejodlowska @sagesharp this is ready for a final round of review.

ewdurbin commented:
@ejodlowska can you provide the final room names/numbers and purpose that we have for PyCon 2018? I think making sure those are up to date in the document and I think that's the last remaining open question.

ejodlowska commented:
Sorry for the delay. Room 23 on May 11 and Room 25C May 12-13.

ewdurbin committed:
generalize

ewdurbin committed:
room numbers

ewdurbin committed:
remove older notes from README

ewdurbin committed:
be consistent

ewdurbin committed:
formatting
